kitsune/settings.py

```diff
@@ -579,16 +581,27 @@ def parse_conn_max_age(value):
     OIDC_ENABLE = False
     ENABLE_ADMIN = False
 else:
-    OIDC_ENABLE = config('OIDC_ENABLE', default=False, cast=bool)
+    OIDC_ENABLE = config('OIDC_ENABLE', default=True, cast=bool)
```

Comment:
Why are you changing the defaults here ?

Reply:
If it's set to readonly then this value if False. Moving forward with FxA it makes sense to have this enabled by default.